packages/feedback-service/package.json

```diff
@@ -36,12 +36,15 @@
   "contributors": [],
   "license": "MIT",
   "dependencies": {
+    "@apollo/federation": "^0.14.1",
     "apollo-log": "^0.1.0",
     "apollo-server-express": "^2.10.1",
     "body-parser": "^1.19.0",
     "clean-webpack-plugin": "^3.0.0",
     "cookie-parser": "^1.4.4",
     "cors": "^2.8.5",
+    "create-jira-ticket-api": "^1.0.1",
```

Comment:
Use the `jira-client` package instead of this one.
https://www.npmjs.com/package/jira-client
It has more downloads and is maintained. 

`create-jira-ticket-api` has not been updated since last 3 years.

Reply:
Removed.